phpstan.neon.dist

```diff
@@ -48,7 +48,27 @@ parameters:
             message: "#^Method Tests\\\\TestCase\\:\\:.*\\(\\) is not final, but since the containing class is abstract, it should be\\.$#"
             count: 2
             path: tests/TestCase.php
-
+        -
+            message: '#^Parameter \#3 \$installationManager of class Composer\\Package\\Locker constructor expects Composer\\Installer\\InstallationManager, Composer\\Repository\\RepositoryManager given.#'
```

Comment:
Sorry about that, I also fixed it in my PR :sweat_smile: https://github.com/nunomaduro/phpinsights/pull/444/files#diff-6f19df6a6307a48db0940e6897591fb08776d2db8a6134737aa708defdb5c92dR51-R54 

You can leave this, we'll manage this clean later :+1: